test/todo/json5.js

```diff
@@ -53,25 +55,35 @@ testCases.forEach((testCase) => {
       const file = fs.readFileSync(testPath, 'utf8');
 
       test.test(testName, (test) => {
-        switch (ext) {
-          case '.json':
-            test.strictSame(jstp.parse(file), JSON.parse(file));
-            break;
-          case '.json5':
-            test.strictSame(jstp.parse(file), extendedEval(file));
-            break;
-          case '.js': {
-            const supportedTests = supportedByUs[testCase.name];
-            if (supportedTests && supportedTests.includes(testName)) {
-              test.strictSame(jstp.parse(file), extendedEval(file));
-            } else {
-              test.throws(() => jstp.parse(file));
-            }
-            break;
+        if (ext === '.json') {
+          let value;
+          test.doesNotThrow(() => value = jstp.parse(file));
```

Comment:
This testing is redundant and hides error message in case when parsing actually throws.